src/app/oxford-debate/page.tsx

```diff
@@ -76,11 +91,7 @@ export default function OxfordDebate() {
             {/* {debate.conf.proTeam ? "as the proposition" : "in favour"} */}
             {aspropo}
           </p>
-          <Dots
-            stages={[0, 2, 4, 6]}
-            stage={stage}
-            flashCurrent={running && !advocem}
-          />
+          <Dots stages={[0, 2, 4, 6]} stage={stage} flashCurrent={running && !advocem} />
```

Comment:
This is aimless, probably due to a code formatter, right? I feel like the prior version was more readable.